met-api/migrations/versions/8ca063aafc01_create_comment_table.py

```diff
@@ -0,0 +1,59 @@
+"""empty message
+
+Revision ID: 8ca063aafc01
+Revises: 1113e0ad66c3
+Create Date: 2022-07-28 15:13:35.368696
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from datetime import datetime
+
+
+# revision identifiers, used by Alembic.
+revision = '8ca063aafc01'
+down_revision = '1113e0ad66c3'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    comment_status_table = op.create_table('comment_status',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('status_name', sa.String(length=50), unique= True, nullable=False),
+    sa.Column('description', sa.String(length=50), nullable=True),
+    sa.Column('created_date', sa.DateTime(), nullable=True),
+    sa.Column('updated_date', sa.DateTime(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('comment',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('text', sa.Text(), nullable=False),
+    sa.Column('submission_date', sa.DateTime(), nullable=True),
+    sa.Column('reviewed_by', sa.String(length=50), nullable=True),
```

Comment:
I think we want a foreign key for the user table (user id) here

Reply:
Added